### ISSUE TITLE
Change icon to customized image

### ISSUE THREAD

**Issue by jlinseman** [JuiceMind/JuiceMind#278]
Replit had the ability to change the Team Icon to something more customized. Can this functionality be added?

ie.
Juicemind: 
![image](https://github.com/user-attachments/assets/ac1797cd-8ec8-499c-b0de-1d707f87e04c)

Replit:

![image](https://github.com/user-attachments/assets/7d90c195-f0b5-4ca2-8212-77c7c1045ff2)

**Comment by juicemind-dev** [comment]
Yes, that's a good idea and we can support this. It likely won't be as urgent so it may take some time for us to release this, is that ok?
